pkgs/by-name/ky/kyverno-chainsaw/package.nix

```
@@ -0,0 +1,39 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "kyverno-chainsaw";
+  version = "0.1.7";
+  cli = "chainsaw";
+
+  src = fetchFromGitHub {
+    owner = "kyverno";
+    repo = "chainsaw";
+    rev = "v${version}";
+    hash = "sha256-v71qAJSpnbHd+jkPkR34IVNvwWLhm04TrPzduB6ZOgA=";
+  };
+
+  vendorHash = "sha256-lHV5Ik/L/Svn9AvcKZupq778ektEsbxfOkCts4Ocx9g=";
+
+  ldflags = [
+    "-s" "-w"
+    "-X github.com/kyverno/chainsaw/pkg/version.BuildVersion=v${version}"
+    "-X github.com/kyverno/chainsaw/pkg/version.BuildHash=${version}"
+    "-X github.com/kyverno/chainsaw/pkg/version.BuildTime=1970-01-01_00:00:00"
+  ];
+
+  doCheck = false; # requires running kubernetes
+
+  meta = with lib; {
+    description = "Chainsaw is a tool developed to run end-to-end tests in Kubernetes clusters.";
```

```suggestion
    description = "Declarative approach to test Kubernetes operators and controllers";
```

The description should not repeat the package name, see contributing guide

---

For my own future reference see section [**Meta attributes** in `pkgs/README.md`](https://github.com/NixOS/nixpkgs/tree/master/pkgs#meta-attributes)